package.json

```diff
@@ -226,14 +227,23 @@
     "**/rc-tree": "^5.7.12"
   },
   "ava": {
-    "files": ["./public-test/test-bundle/**/*.{js,jsx}"],
-    "ignoredByWatcher": ["./binaryData/**/*.*"],
-    "require": ["./frontend/javascripts/test/_ava_polyfill_provider.ts"],
+    "files": [
+      "./public-test/test-bundle/**/*.{js,jsx}"
+    ],
+    "ignoredByWatcher": [
+      "./binaryData/**/*.*"
+    ],
+    "require": [
+      "./frontend/javascripts/test/_ava_polyfill_provider.ts"
+    ],
     "snapshotDir": "frontend/javascripts/test/snapshots",
     "concurrency": 8
   },
   "c8": {
-    "exclude": ["public-test/test-bundle/test/**/*.*", "frontend/javascripts/test/**/*.*"],
+    "exclude": [
+      "public-test/test-bundle/test/**/*.*",
+      "frontend/javascripts/test/**/*.*"
+    ],
```

Comment:
TODO: Reformat this file :)

Reply:
you mean I should undo the formatting? sublime formats this with biome automatically. maybe we should extend the check-frontend script so that it does this, too?